src/Symfony/Component/Routing/Exception/MissingMandatoryParametersException.php

```diff
@@ -19,4 +19,36 @@
  */
 class MissingMandatoryParametersException extends \InvalidArgumentException implements ExceptionInterface
 {
+    private array $missingParameters = [];
+    private string $routeName = '';
+
+    public function __construct(/*array*/ $missingParameters = [], /*string*/ $routeName = '', /*int*/ $code = 0, \Throwable $previous = null)
+    {
+        if (\is_array($this->missingParameters)) {
```

Comment:
```suggestion
        if (\is_array($missingParameters)) {
```
you should check the argument, right?

Reply:
Indeed, thank you for catching that !
I added a test to prove the legacy usage of this exception.